### DISCUSSION TITLE
Alphanumeric seeds

### DISCUSSION THREAD

**Discussion by veradalelabs** [jacobsjo/mc-datapack-map#35] [Q&A] [answered]
Is it possible to use an alphanumeric value for the world seed?

I can type something in, but it just reverts to zero.

Not sure if this is an issue, or working as intended.

**Comment by jacobsjo** [comment, marked as answer]
Thats currently not supported. I've created an issue in #36 to track that. For now, simply run `/seed` in a minecraft world after creation to get the numeric seed.
